appium/webdriver/extensions/android/network.py

```diff
@@ -55,6 +69,25 @@ def toggle_wifi(self):
         self.execute(Command.TOGGLE_WIFI, {})
         return self
 
+    def set_network_speed(self, speed_type):
+        """Set the network speed emulation.
+        Android Emulator only.
+
+        :Args:
+         - speed_type (str): The network speed type.
+           A member of the const appium.webdriver.extensions.android.network.NetSpeed.
```

Comment:
I think we could use better approach to make method cross-references: https://stackoverflow.com/questions/21289806/link-to-class-method-in-python-docstring

Reply:
Yes, we need to do so. 
(I did so only at this time..)
 https://github.com/appium/python-client/blob/af2fa7034fe3874e666316ce701ce25e8e64425b/appium/webdriver/extensions/device_time.py#L34